README.md

```diff
@@ -124,7 +124,8 @@ with the CSI driver pre-installed and no further steps are required.
 
 **Requirements:**
 
-* `--allow-privileged` flag must be set to true for both the API server and the kubelet
+* `--allow-privileged` flag must be set to true for the API server
+* `--allow-privileged` flag must be set to true for the kubelet in Kubernetes 1.14 and below, and not be set otherwise
```

Comment:
`and not be set otherwise` makes it sound a bit like setting it will cause problems specifically for our CSI driver, whereas actually the issue is that it doesn't exist. I would probably just call out here that the flag is removed in later releases, something like `... in Kubernetes 1.14 and below (flag does not exist in later releases)`.

Reply:
Good suggestion, done. I might adjust a wording a bit more if it turns out that the API server flag might not be needed, either generally or in specific versions. (See main discussion on this PR.)